src/redux/reducers/index.js

```diff
@@ -62,16 +62,11 @@ export const resourceTypeFiltersReducer = (state = preloadResourceTypeFilters, a
   switch (action.type) {
     case actionTypes.CREATE_COLLECTION:
     case actionTypes.SELECT_COLLECTION:
+    case actionTypes.DELETE_COLLECTION:
+    case actionTypes.CLEAR_COLLECTION:
```

Comment:
this is cleanup from the pivot to consolidating `CollectionPreview` with `CatalogScreen`